src/utils/content-type.ts

```diff
@@ -10,14 +19,14 @@ import mime from 'mime-types';
  * @param {(string | Buffer)} content the content to check
  * @returns {(string | undefined)} a valid content type or undefined
```

Comment:
The TypeScript types now say that this returns a `Promise<string | undefined>`, should update the docs/comments to reflect that.